app/views/locations/descriptions/_form.html.erb

```diff
@@ -1,30 +1,40 @@
-<%= form_with(scope: :description, url: action,
+<%
```

Comment:
Would it make sense to move this to a helper?

Reply:
Joe, you're right. Actually I think this would be a proper use of a presenter, rather than a helper. The difference as far as I can tell is that presenters are designed to just perform logic and return a set of data, where helpers return HTML markup.

I don't know if you noticed my recent PR where I cleaned up the matrix box presenter and thumbnail presenter to bring them back to the intended purpose, and as a result they're less confusing. (that PR was only around briefly, I merged immediately).

I don't picture this getting more complex, so i'm ok with it like this, but let's try a new presenter. It will be good practice to show the pattern.

Reply:
Presenters are a good example of "service objects".

You initialize them with a few args, and they give you an object full of data. Simple as that. 
An advantage is that they do not pollute the global namespace for methods, like helpers do.

Check the three we have for examples. I think the old `RssLogsPresenter` scared everyone off them, because it was generating all kinds of HTML through very convoluted means. 

The key is to keep html generation accessible in helpers, and just use presenters to figure out logic that's not coming straight off the db. They can lighten the load for controllers and remove logic from views.